content/docs/04.5-devx/03-cluster-groups.md

```diff
@@ -101,6 +101,61 @@ To delete a cluster group, select `Delete Cluster form the **Settings** and ente
 * Azure
 * AKS
 * VMWare
+* OpenShift
+
+<Tabs>
+<Tabs.TabPane tab="Configure OpenShift" key="Configure OpenShift">
+
+#### Configure OpenShift 
+
+<br />
+
+To deploy a virtual cluster on OpenShift:
+
+
+1. Create a new Cluster Group or edit an existing one and click **Settings**.
+2. Select **Settings** in the **Cluster Group Settings** pane.
+3. In the **Advanced Config** file, locate the section ``#Configure SecurityContext of the containers in the VCluster pod``.
+
+4. Comment out these lines: 
+
+    * ``fsGroup``
+    * ``runAsGroup``
+    * ``runAsUser``
+    
+4. Set ``openshift.enable: true``.  
+
+5. Verify these default parameter values are set as follows:   
+
+    * ``allowPrivilegeEscalation: false``
+    * ``capabilities.drop: [-all]``
```

Comment:
@ritawatson it should be `capabilities.drop: [all]`, not `capabilities.drop: [-all]`

Reply:
@TylerGillson I think I get it. The yaml shows -all, but this is probably just one item in a list of potential other values. That's why I included the dash.